lib/lambda_ethereum_consensus/store/block_store.ex

```diff
@@ -8,13 +8,14 @@ defmodule LambdaEthereumConsensus.Store.BlockStore do
   @block_prefix "block"
   @blockslot_prefix @block_prefix <> "slot"
 
-  @spec store_block(SszTypes.BeaconBlock.t()) :: :ok
-  def store_block(%SszTypes.BeaconBlock{} = block) do
+  @spec store_block(SszTypes.SignedBeaconBlock.t()) :: :ok
+  def store_block(%SszTypes.SignedBeaconBlock{} = signed_block) do
+    block = signed_block.message
     {:ok, block_root} = Ssz.hash_tree_root(block)
-    {:ok, encoded_block} = Ssz.to_ssz(block)
+    {:ok, encoded_signed_block} = Ssz.to_ssz(signed_block)
 
     key = block_key(block_root)
-    Db.put(key, encoded_block)
+    Db.put(key, encoded_signed_block)
 
     # WARN: this overrides any previous mapping for the same slot
     # TODO: this should apply fork-choice if not applied elsewhere
```

Comment:
I think `block_root_by_slot_key(block.slot)` should change to `block_root_by_slot_key(block.message.slot)`.

Reply:
But block is not a signed block in this case, see line 13.